dapp_user/infrastructure/repositories/user_repository.py

```diff
@@ -46,3 +46,15 @@ def get_user_preferences(self, user_row_id):
     def delete_user(self, username):
         query = "DELETE FROM user WHERE username = %s "
         self._repo.execute(query, [username])
+
+    def register_user_data(self, user):
+        """ register user data """
+        if not user.email_verified:
+            raise Exception("Email verification is pending.")
+        query_parameters = [user.email, "", user.origin, user.name, user.email, user.email_verified,
```

Comment:
When are we updating the email verification flag?

Reply:
We are not updating once inserted. As from starting we have email verified as true.